src/utils/fees.js

```diff
@@ -10,7 +10,8 @@ const SEND_FEE_UNITS = {
   BNB: 21000,
   NEAR: 10000000000000,
   MATIC: 21000,
-  ERC20: 90000
+  ERC20: 90000,
+  ARBETH: 2000000
```

Comment:
`2000000` gas for sending in arbitrum? Is this correct?
Seems to be the same as initiating a swap - doesn't seem right

Reply:
That was suggestion from the tech team to use slightly higher gas limit, because the gas price could vary and I left it for safety reasons. 

But from my observations it is taking around   609500 Gas, so I can put it to 700 000